### PR TITLE
Additional option "keepResults"

### PR DESCRIPTION
**Pull request by maettyhawk**
#127

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,6 @@
 node_modules
 coverage
 *.log
-lib
```
Comment (smeijer) [inline .gitignore]: please don't include the lib folder in your commit.

**Comment by smeijer** [comment]
The change is looking great; only you have committed the lib folder. Can you please undo this?

It would also be great if you can add this option to the `readme`.

I'll merge as soon as `lib` problem has been fixed. Adding to the `readme` would be a nice bonus.

**Comment by maettyhawk** [comment]
Should be fine now.

**Comment by smeijer** [comment]
Awesome. Thanks!

**Pull request merged**